warehouse/query-core/src/main/java/datawave/query/index/lookup/RangeStream.java

```diff
@@ -496,26 +493,19 @@ public IndexStream visit(ASTAndNode node, Object data) {
                 return ScannerStream.noOp(node);
 
             } else if (builder.size() == 0) {
-                return ScannerStream.unindexed(node);
+                return ScannerStream.noData(node);
             } else {
                 Intersection build = builder.build(executor);
                 switch (build.context()) {
                     case ABSENT:
                         return ScannerStream.noData(build.currentNode(), build);
-                    case IGNORED:
-                        return ScannerStream.ignored(build.currentNode(), build);
-                    case EXCEEDED_TERM_THRESHOLD:
-                    case EXCEEDED_VALUE_THRESHOLD:
                     case PRESENT:
                     case VARIABLE:
+                    case DELAYED:
                         return build;
-                    case UNINDEXED:
-                        return ScannerStream.unindexed(build.currentNode(), build);
-                    case UNKNOWN_FIELD:
                     case INITIALIZED:
-                        return ScannerStream.unknownField(build.currentNode(), build);
                     default:
-                        return ScannerStream.unknownField(node, build);
+                        throw new RuntimeException("unhandled stream context: " + build.context());
```

Comment:
to confirm intent, INITIALIZED at this point is an error condition?

Reply:
Yes, at this point the concurrent scanner initializer has run so if the intersection is still in the initialized state then something went very wrong.